packages/@aws-cdk/aws-codecommit/package.json

```diff
@@ -116,7 +116,8 @@
       "docs-public-apis:@aws-cdk/aws-codecommit.RepositoryEventTrigger.ALL",
       "docs-public-apis:@aws-cdk/aws-codecommit.RepositoryEventTrigger.UPDATE_REF",
       "docs-public-apis:@aws-cdk/aws-codecommit.RepositoryEventTrigger.CREATE_REF",
-      "docs-public-apis:@aws-cdk/aws-codecommit.RepositoryEventTrigger.DELETE_REF"
+      "docs-public-apis:@aws-cdk/aws-codecommit.RepositoryEventTrigger.DELETE_REF",
+      "attribute-tag:@aws-cdk/aws-codecommit.Repository.repositoryCloneUrlGrc"
```

Comment:
Hmmm... what is the error without this line?

If it _is_ required, can you move it a little bit up, above the big block of `docs-public-apis` exclusions, so that the diff is nicer? Thanks! 🙂

Reply:
`attribute properties must have an "@attribute" doctag`

but this new `repositoryCloneUrlGrc` is not a CF attribute.